content/blog/2024-01-22.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "2024-01-22"
+date = 2024-01-22
++++
+
+---
+
+Today is Monday, day 022 of 366 (6.01%). We're in week 04 of 52 (7.69%).
+
+## Planning
+
+What I am going to do today:
+
+- [ ] configure [edit snippet page](https://github.com/OmnicodeSolutions/luisa_drf_flutter_client/blob/main/lib/edit_snippet.dart)'s save button
+- [x] fix logout button
+
+## Activities executed
+
+Today I fixed the logout button, I believe the env file just needed some time to load, because when I ran the app today it didn't give that same error, but there was a new issue: the rout that was leading to the login page after logout was allowing the user to go back to the previous state. I looked for a way to disable flutter's back button, and found [this](https://flutterdesk.com/disable-or-override-back-button-in-flutter/), which took me some time to be able to implement in the project, but in the end what I wanted was way more simple: all I had to do was set the `automaticallyImplyLeading` property from the [AppBar](https://api.flutter.dev/flutter/material/AppBar-class.html) to false. 
```

Comment:
```suggestion
Today I fixed the logout button, I believe the env file just needed some time to load, because when I ran the app today it didn't give that same error, but there was a new issue: the route that was leading to the login page after logout was allowing the user to go back to the previous state. I looked for a way to disable flutter's back button, and found [this](https://flutterdesk.com/disable-or-override-back-button-in-flutter/), which took me some time to be able to implement in the project, but in the end what I wanted was way more simple: all I had to do was set the `automaticallyImplyLeading` property from the [AppBar](https://api.flutter.dev/flutter/material/AppBar-class.html) to false. 
```